frameworks/elastic/README.md

```diff
@@ -218,33 +215,31 @@ docker run mesosphere/janitor /janitor.py -r elastic-role -p elastic-principal -
-- Serial vs Parallel deployment. By default, the Elastic framework tells Mesos to install/update everything in parallel. You can change this to serial to have each node installed one at a time.
+- Node counts: At least 1 data node is required for the cluster to operate at all. You do not need to use a coordinator node unless you are using Kibana. There is no maximum for node counts.
+- Ports: You can pick whichever ports work for your DC/OS cluster. The defaults have master nodes listening on 9200, data nodes on 9201, ingest on 9202, and coordinators on 9203 for HTTP. For transport port numbers, the defaults are the HTTP port number + 100 (i.e., 9300, 9301, 9302, and 9303). If you want each Elasticsearch node to run on a different agent and never be collocated with any other elasticsearch nodes in the same cluster, specify the same HTTP and transport port values for each node type. If you want multiple master nodes from different clusters on the same host, specify different master HTTP and transport ports for each cluster.
+- Serial vs Parallel deployment. By default, the Elastic framework tells Mesos to install and update everything in parallel. You can change this to serial in order to have each node installed one at a time.
 
-It can be confusing to understand what parts of the Elasticsearch cluster can be modified through the Mesosphere DC/OS framework at runtime, what gets specified initially and is immutable, and what gets modified directly through the Elasticsearch cluster update settings API. The most important settings are the immutable ones, so let’s start with those.
+It can be confusing to understand which parts of the Elasticsearch cluster can be modified through the Mesosphere DC/OS framework at runtime, what gets specified initially and is immutable, and what gets modified directly through the Elasticsearch cluster update settings API. The most important settings are the immutable ones, so let’s start with those.
 
-### Immutable settings (at cluster creation time via Elastic package UI or JSON options file via CLI)
+### Immutable settings (at cluster creation time via Elasticsearch package UI or JSON options file via CLI)
 
 - Service name (aka cluster name). Can be hyphenated, but not underscored.
-- Master transport port
-- Disk sizes/types
+- Master transport port.
+- Disk sizes/types.
 - Modifiable settings (at runtime via Marathon env vars):
```

Comment:
I got a little confused here -- are all the settings that follow modifiable? Can we differentiate between those modifiable via the API vs from DC/OS?